linux-mingw-w64.cmake

```diff
@@ -23,6 +23,4 @@ set(CROSS_COMPILE_TOOLCHAIN_PREFIX "x86_64-w64-mingw32")
 include(
     "${CMAKE_CURRENT_LIST_DIR}/compiler/gcc-cross-compile-simple-layout.cmake"
 )
-include("${CMAKE_CURRENT_LIST_DIR}/flags/cxx11.cmake")
```

Comment:
I don't think changing functionality of existent toolchain is a good idea. Also C++11 is kind of a default standard in this collection. I think it's better to leave this code and just name new toolchain `linux-mingw-w64-cxx98.cmake`